Themis/Models/Authentication.swift

```diff
@@ -18,64 +18,37 @@ class Authentication: NSObject {
 
     static var shared: Authentication!
 
-    enum AuthenticationError: Error {
-        case tokenNotFound
-    }
-
-    @objc dynamic var token: String?
-    let keychain: Keychain
+    let url: URL
 
-    override init() {
-        keychain = Keychain(service: "feedback2go.auth") // should this be changed as well?
```

Comment:
So we are removing Keychain support completely? Maybe we could think about whether we could store the cookie in the Keychain at some later point?

Reply:
No need to save the cookie in the keychain as it is stored int the URLSession.